elastic/kibana
# [ML] Add test to delete job

## Summary

This PR adds a test to delete a anomaly detection job.

### High level test steps

- This test starts where the `clone single metric job` test left off
- Check that the clone job has job results before deletion
- Delete the clone job
- Check that the job isn't displayed in the job list any more
- Check that there are results for the deleted job any more
